analysis/wwz/make_datacards.py

```diff
@@ -286,7 +312,14 @@ def get_syst_base_name_lst(in_lst):
 
                 # Handle negative cases
                 if (valvar_kappa_up[0]<=0) and (valvar_kappa_do[0]<=0):
-                    raise Exception(f"Both Kappas Neagtive for process: {proc}, category: {cat}, systematic: {sys}")
+                    if (set(yrs_lst) == set(["2022","2022EE","2023","2023BPix"])) and (cat=="sr_4l_bdt_of_1") and (sys=="FSR") and (proc=="ttZ"):
```

Comment:
Is this the only case as of now? I assume it is.

Reply:
Yes only case for now.